website/docs/open-source/getting-started/deploy-OSS.mdx

```diff
@@ -1,9 +1,9 @@
 ---
-title: 2. Deploy an Application
+title: Step 3 - Deploy an Application
 hide_title: true
 ---
 
-# Part 2: Deploy an Application
+# Step 3: Deploy an Application
```

Comment:
This looks odd (there's no Step 1) but I'm assuming it's here in anticipation of more changes to come.

Reply:
Do you mean in the sidebar view, there's no Step 1? If we set "Weave Gitops Open Source" as a link collapse button instead of as a page, then the Step 1: Install...page will appear next to Steps 2 and 3. Thoughts?

Reply:
Yes that might work. Currently it looks like this 

<img width="412" alt="Screenshot 2023-06-05 at 20 01 53" src="https://github.com/weaveworks/weave-gitops/assets/8741709/c6165dda-b216-44c9-816f-d0a8a9ff7e68">

Reply:
Yes, I saw this and pinged Luiz about it. If we could make it a link collapse button it would be better.